share/spice/playing_cards/content.handlebars

```diff
@@ -0,0 +1,5 @@
+<div>
```

Comment:
There's a much better way of doing this :)

Our templates create all the markup for you if you disable the `detail` pane and use a `normalize` function to indicate the `image` for each tile.

Take a look at the Movies or InTheaters IA's to see how they do this. You shouldn't need any custom handlebars templates :)